### DISCUSSION TITLE
Resume Bundle  X  ATS Scoring

### DISCUSSION THREAD

**Discussion by ImmidiSivani** [recodehive/resume-pitch#95] [Q&A]
Name : Immidi Sivani
Linkedin Link: https://www.linkedin.com/in/sivani-immidi-0595ab257/
Resume : 
[sivani_final.pdf](https://github.com/user-attachments/files/17008848/sivani_final.pdf)
Github Link: https://github.com/ImmidiSivani
I am a 3rd-year undergraduate student looking for full stack engineer/software engineer roles

**Comment by sanjay-kv** [comment]
[sivani.pdf](https://github.com/user-attachments/files/17089380/sivani.pdf)
[ATS Version of CV (25).docx](https://github.com/user-attachments/files/17089383/ATS.Version.of.CV.25.docx)

@ImmidiSivani
